hedera-node/configuration/testnet/settings.txt

```diff
@@ -1,33 +1,47 @@
-checkSignedStateFromDisk,                      1
-csvFileName,                                   MainNetStats
-csvOutputFolder,                               data/stats
-dbBackup.active,                               1
-dbConnection.active,                           1
-dbRestore.active,                              1
+                                          #############################
+                                          #    Gossip / Networking    #
+                                          #############################
+
+chatter.useChatter,                            false
 doUpnp,                                        false
-enableEventStreaming,                          true
-eventsLogDir,                                  /opt/hgcapp/eventsStreams
-eventsLogPeriod,                               5
-lockLogTimeout,                                4000
-maxEventQueueForCons,                          1000
-maxOutgoingSyncs,                              4
+maxOutgoingSyncs,                              8
```

Comment:
This value should not be changed to 4

Reply:
Reverted.

Is this because of the size difference between networks?

Reply:
yes